Gemfile

```diff
@@ -15,8 +15,6 @@ gem "bootsnap", "~> 1.4"
 
 gem "puma", ">= 5.6.2"
 
-gem "faker", "~> 2.14"
```

Comment:
This change and the introduction of faker in decidim-dev is a manual change made so the `Decidim::GemManager#patch_component_gemfile_generator` didn't raise an exception (as faker isn't available in decidim-dev gemspec in v0.27).

Related to #10867 and #12250